packages/mobile/src/analytics/Events.tsx

```diff
@@ -288,6 +288,10 @@ export enum CeloExchangeEvents {
   celo_withdraw_error = 'celo_withdraw_error', // when there's an error on the withdrawal transaction
 }
 
+export enum FiatExchangeEvents {
```

Comment:
I think this should also be added to `AnalyticsEventType` at the bottom of this file

Reply:
Good catch, updated